test/compiler_spec.js

```diff
@@ -29,8 +29,9 @@ describe('Vue component compiler', function () {
   test('empty')
   test('less')
   test('sass')
-  test('myth')
+  // test('myth')
```

Comment:
This is a great change, thanks!
But is this a mistake? Why do you remove myth testing? :)